docs/prepping-local-system-runbook-linux.md

```diff
@@ -1,63 +1,58 @@
+To deploy Pega Platform using a local Linux system on which you can run commands with administrator privileges, prepare your system with required applications and configuration files that you need for your deployment. By preparing your system, you can complete the deployment without having to pause to obtain a Linux executable file or prepare a configuration file that is required to complete the deployment.
 
-Assumptions and prerequisites
------------------------------
+Before you begin
+----------------
 
-This guide assumes:
+To prepare a Linux system, it is assumed:
 
 - You have a basic familiarity with running commands from a Linux command prompt with and without root privileges.
 
 - You use a packaging tool to install application packages. For demonstration purposes, this document refers to the Ubuntu Advanced Packaging Tool (apt) commands that is available for use on Ubuntu Linux distributions.
 
-- Basic familiarity with GitHub account with which you will download a Pega-managed GitHub repository containing configuration files and scripts that you use to install Pega Platform and then deploy it in the Kubernetes cluster.
+- You have basic familiarity with a GitHub account with which you download a Pega-managed GitHub repository that contains configuration files and scripts that support installing Pega Platform and deploying it in the Kubernetes cluster.
 
 Creating a local folder to access all of the configuration files
 ----------------------------------------------------------------
 
-Deploying with Helm requires that you run commands from a specific folder on your local system. To ensure you stay oriented to the correct filepath, these instructions always use the reference \<local filepath\>/\<platform\>-demo directory when you must extract files to a folder or run commands from a folder.
+Deploying with Helm requires you to run commands from a specific folder on your local system. To ensure you use the correct filepath, these instructions always use the reference `<local filepath>/<platform>-demo` folder when you extract files to a folder or run commands from a folder.
 
-In order to stay consistent with the instructions, you can create a directory called \<platform\>-demo on your local system in your /home directory. This way, you associate the generic \<local filepath\>/\<platform\>-demo references to the folder /home/\<*linux-username*\>/\<platform\>-demo that is specific to your local
-system.
+To stay consistent with the instructions, you can create a folder named `<platform>-demo` on your local system at the top level of your Windows user folder. This way, you associate the generic `<local filepath>/<platform>-demo` references to the folder `/home/<linux-username>/<platform>-demo` that is specific to your local system.
 
 To create this folder, open a Linux command prompt and enter:
 
 `$ mkdir /home/<linux-username>/<platform>-demo`
```

Comment:
Instead of `/home/`, can we use `~/` so that it will work if a home directory is elsewhere?